src/redux/books/Books.js

```diff
@@ -0,0 +1,36 @@
+import { v4 as uuidv4 } from 'uuid';
+
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] According to the project requirement, 🤔  from this step, you should configure the Redux Store and write actions and reducer for adding and removing books but so far I can not nether able to add or remove any book from the list, Kindly make this 👍🏻

Reply:
Well, I think we are supposed to implement the `actions` and `store` in different files :)
I've created the actions in the `books.js` file and placed the `store` in `configureStore.js` 
for this round that's all we are asked to implement as a requirement.